.github/workflows/scan_images.yaml

```
@@ -0,0 +1,20 @@
+name: Scan images
+
+on:  
+  schedule:
+    - cron: '00 23 * * *'
+  workflow_dispatch:
+
+jobs:
+
+  scan-images:
+    name: Scan published images and report vulnerabilities
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/get-published-images-scan-and-report.yaml@main
+    strategy:
+      matrix:
+        branch: [main, track/2.27]
```

We do want only the version that is actually deployed:
```suggestion
        branch: [track/2.27]
```